src/main/java/org/takes/facets/fork/MediaTypes.java

```diff
@@ -132,7 +132,8 @@ public boolean isEmpty() {
     @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
     private static SortedSet<MediaType> parse(final String text) {
         final SortedSet<MediaType> list = new TreeSet<MediaType>();
-        for (final String name : text.toLowerCase(Locale.ENGLISH).split(",")) {
+        final String[] split = new EnglishLowerCase(text).string().split(",");
```

Comment:
@dmzaytsev We can't... because the [LineLength](http://checkstyle.sourceforge.net/config_sizes.html#LineLength) police (will exceed 80).